website/source/docs/config/index.html.md

```diff
@@ -125,6 +127,34 @@ The supported options are:
       are generally considered less secure; avoid using these if
       possible.
 
+### Connecting to Vault Enterprise in HashiCorp Atlas
+
+Adding an "atlas" block will initiate a long-running connection to the
+[SCADA](https://scada.hashicorp.com) service. The SCADA connection allows the
+Vault Enterprise interface to securely communicate with and operate on your
+Vault cluster.
+
+The "atlas" `listener` supports these options:
+
+  * `endpoint` (optional) - The endpoint address used for Vault Enterprise interface
+      integration. Defaults to the public Vault Enterprise endpoints on Atlas.
+
+  * `infrastructure` (required) - Used to provide the Atlas infrastructure name and
+      the SCADA connection. The format of this is `username/environment`.
+
+  * `node_id` (required) - The identifier for an individual node—used in
+      the Vault Enterprise dashboard.
+
+  * `token` (required) - A token from Atlas used to authenticate SCADA session. Generate
+      one in the [Atlas](https://atlas.hashicorp.com/settings/tokens).
+
+Additionally, the global `cluster_name` will be used to identify your cluster
```

Comment:
how is `cluster_name` set? 

also to make sure I understand the hierarchy it is:
An infrastructure is a group of clusters. A cluster is a group of nodes.